candi/bashible/common-steps/all/098_update_node_annotations.sh.tpl

```diff
@@ -27,3 +61,27 @@ fi
   version of the registry configuration is currently applied on the node.
 */}}
 bb-kubectl-exec annotate node $(bb-d8-node-name) registry.deckhouse.io/version={{ .registry.version | quote }} --overwrite
+
+# check if d8-dhctl-converger user exists and annotate node
+
+CONVERGER_USER_ANNOTATION=$(bb-kubectl-exec --kubeconfig=/etc/kubernetes/kubelet.conf get no "$D8_NODE_HOSTNAME" -o json |jq -r '.metadata.annotations."node.deckhouse.io/has-converger-nodeuser"')
```

Comment:
let to add this check for add/remove annotation for d8-converge-user
https://github.com/deckhouse/deckhouse/blob/8687a0d0fed879b71aa0628f65396a32eae3463d/candi/bashible/common-steps/all/005_integrate_kubernetes_data_device.sh.tpl#L17